templates/frontend/objects/announcement_summary.tpl

```diff
@@ -14,25 +14,25 @@
 	{assign var="heading" value="h2"}
 {/if}
 
-<article class="obj_announcement_summary">
-	<{$heading} class="announcements">
-		<a href="{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()}">
-			{$announcement->getLocalizedTitle()|escape}
-		</a>
-	</{$heading}>
-	<div class="announcements date text-muted small">
-		{$announcement->getDatePosted()|date_format:$dateFormatShort}
-	</div>
-	<div class="announcements summary">
-		{$announcement->getLocalizedDescriptionShort()|strip_unsafe_html}
-
-	</div>
-	<a href="{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()}" class="announcements btn btn-secondary">
-		<span>
-			{translate key="common.readMore"}
-		</span>
-		<span class="sr-only">
-			{translate key="common.readMoreWithTitle" title=$announcement->getLocalizedTitle()|escape}
-		</span>
+<{$heading}>
```

Comment:
It looks like there is no longer a root element, like `<article>` that each summary is in. This can be ok, but it means every time you call the template you need to provide a root element:

```
{foreach from=$announcements item=$announcement}
  <article>
    <include file="../announcement_summary.tpl">
  </article>
{/foreach}
```

Typically, for other objects, we include the root element in the template. That's just a convention though and can be broken if this is preferred. I'm just mentioning it for you to decide.